packages/config/src/config/job.ts

```diff
@@ -0,0 +1,15 @@
+import { BaseConfig } from './base.js';
+
+export enum JobStatus {
+  Processing = 'processing',
+  Complete = 'complete',
+  Fail = 'fail',
+}
+
+export interface ConfigJob extends BaseConfig {
+  /** Job Status for the imagery importing batch jobs */
+  status: JobStatus;
+
+  /** Job Batch processing error messages */
+  error?: string;
```

Comment:
error should be on a seperate type, we should have something like

```
export interface ConfigJobFialed  {
 status: 'failed'
 error: ...
}

export interface ConfigJob = ConfigJobFailed | ConfigJobSuccess
```
 

then the success option could have extra keys